packages/backend-common/package.json

```diff
@@ -78,6 +78,7 @@
     "morgan": "^1.10.0",
     "node-fetch": "^2.6.7",
     "node-forge": "^1.3.1",
+    "pg": "^8.9.0",
```

Comment:
Don't think we need this dep either, as it should be fine with just the `@types/pg` dependency.